widgets/src/amcharts/charts/Scatter.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+(function(root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3/d3", "./CommonXY", "amcharts.xy", "../../chart/INDChart", "css!./Bar"], factory);
+    } else {
+        root.Scatter = factory(root.d3, root.CommonXY, root.amcharts, root.INDChart);
```

Comment:
Should be:  root.amcharts_Scatter = factory(root.d3, root.amcharts_CommonXY, root.*amcharts*, root.chart_INDChart);

I am not sure what the correct root is for amcharts - you would need to check the docs/samples...